src/nccl_ofi_ep_addr_list.c

```diff
@@ -134,6 +134,7 @@ int nccl_ofi_ep_addr_list_insert(nccl_ofi_ep_addr_list_t *ep_list,
 				 size_t addr_size)
 {
 	int ret = 0;
+	ep_pair_list_elem_t *new_pair;
```

Comment:
Would prefer we initialize these (here, to NULL). In large functions, I've found it's way too easy to use these variables uninitialized (which is why we shouldn't have large functions, but that's a refactoring for another day).

Reply:
Sure, I can fix that.